catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImpl.java

```diff
@@ -163,18 +203,26 @@ public String toString() {
    *
    * @return query request
    */
-  private QueryRequest createQueryRequestForAllWorkspaceMetacards() {
-    return createQueryRequest(filterService.buildWorkspaceTagFilter());
-  }
+  private QueryRequest createQueryRequestForSubscribedToWorkspaces() {
+    Set<String> ids = new HashSet<>();
+    try {
+      List<Map<String, Object>> subscriptions =
+          persistentStore.get(SubscriptionsPersistentStore.SUBSCRIPTIONS_TYPE);
+      ids =
```

Comment:
✏️ This doesn't need to be inside the `try/catch` block. Consider moving the `persistentStore.get()` call and error handling to a separate function so this one can be simplified.

Reply:
refactored